src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -638,6 +639,14 @@ def create(
             logger.info(f"Account group {group.display_name}: {cfg.host}/users/groups/{group.id}/members")
         else:
             logger.info(f"Workspace group {group.display_name}: {cfg.host}#setting/accounts/groups/{group.id}")
+
+        @retried(on=[NotFound], timeout=timedelta(minutes=1))
```

Comment:
@nkvuong : What do you think a reasonable time should be? One, two, five minutes?

Reply:
Two minutes should be enough for most situations